docs/metrics-events.md

```diff
@@ -141,7 +141,6 @@ contains the following fields:
 |`type`|The event name.|
 |`uid`|The user id. An opaque token, HMACed to avoid correlation back to FxA user db.|
 |`locale`|The user's locale. For cases where we aren't localised in their favoured locale(s), the value will be `en-US.default`|
-|`export_date`|The date that the event was exported to S3 by the metrics pipeline.|
```

Comment:
This got removed in a recent refactoring of our import scripts, in order to make the the flow event and activity event tables homegeneous. It can be worked out using `timestamp` anyway, the table where it is useful is `flow_metadata` (which still has it).